composer.json

```diff
@@ -1,68 +1,66 @@
 {
-    "name": "league/oauth2-client",
```

Comment:
Please don't modify the indentation of the composer.json file. Thanks!

Reply:
> 

IDE issues.
Should be fixed now.